### PR TITLE
Gis data

### PR DESCRIPTION
**Pull request by AlexNeary**
I have uploaded the Python Script, as well as some documentation. 
I've also edited the docs/index.md file to include the new documentation.

### CODE REVIEW

File: docs/Tools/GIStoGEXF.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Nodes
+   * PSAP(Public Answering Service Points) just contain the Display Name for the PSAP
+
+   * Fire, Ems, and Law nodes all contain the NG911 unique identifier, along with the display name and a representative point that falls somewhere near the middle of the boundary.
+
+   * Caller Region nodes contain the display name of the PSAP they are linked to, along with a list of points representing squares that make up the area covered by the region. There is only one caller region per PSAP.
+## File Input/Output
+GeoPandas can read several different GIS file formats. `.shp` and `.gpkg` are two of the most common file formats. 
+
+The script is currently looking for fice different layers, PSAP Layer, Law Layer, EMS Layer, Fire Layer, and Provisioning Layer. These layers should be placed in the `GIS Data/Layers/` directory in the GIStoGEXF directory.
```
Comment (jardiamj) [inline docs/Tools/GIStoGEXF.md:23]: Typo: fice instead of five

File: docs/index.md [hunk 2]
```diff
@@ -57,6 +57,10 @@
 
 Here are some [recommended resources](Resources.md) to browse
 
+## Tools
+
+Here is documentation on the [GIS to GEXF](Tools/GIStoGEXF.md) tool
```
Comment (jardiamj) [inline docs/index.md:62]: Maybe write what the purpose of the script within the 911 simulation project is. That way people won't be puzzled by it.

**Review by jardiamj — COMMENTED** [review]
Excellent work Alex! I just left a few comments.
I haven't tested the script yet but I will do that this weekend.

File: Tools/GIS to GEXF tool/GIStoGEXF.py [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    # kc_ems.plot()
+    # kc_psap.plot()
+    # kc_law.plot()
+    # kc_fire.plot()
+    # plt.show()
+
+    # Loop through all EMS boundaries, adding nodes and representative points
+    for w in range(kc_ems.shape[0]):
+        # Find a representative point for the boundary, and convert to string form
+        pt = kc_ems.iloc[w].geometry.representative_point()
```
[the remark this answers is not in the document]
Comment (AlexNeary) [reply Tools/GIS to GEXF tool/GIStoGEXF.py:95]: A representative point is  a cheaply calculated point that lies near the middle of the object. Unlike a centroid, the representative point is guaranteed to lie within the boundaries of the object. This is the best explanation I could fond on how it is calculated. https://gis.stackexchange.com/questions/414260/how-does-geopandas-representative-point-work

File: docs/Tools/GIStoGEXF.md (hunk 1 above)
Comment (AlexNeary) [reply docs/Tools/GIStoGEXF.md]: Fixed on a commit.

File: Tools/GIS to GEXF tool/GIStoGEXF.py [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+        for m in range(kc_psap.shape[0]):
+            if(kc_psap.iloc[m].DsplayName == d_names[n]):
+                geo = kc_psap.iloc[m].geometry
+                polygons.append(geo)
+        new_poly = unary_union(polygons)
+        polys.append(new_poly)
+        names.append(d_names[n])
+
+    # Create a dictionary from the two lists, to be turned into a new GeoDataFrame
+    d = {'DisplayName': names, 'geometry': polys}
```
[the remark this answers is not in the document]
Comment (AlexNeary) [reply Tools/GIS to GEXF tool/GIStoGEXF.py:65]: I think that should work. I'll have to look tomorrow with a fresh brain.

[the remark this answers is not in the document]
Comment (AlexNeary) [reply Tools/GIS to GEXF tool/GIStoGEXF.py]: I was able to get rid of the nested loops. I still had to do a short loop after to merge the polygons using unary_union().

**Review by jardiamj — APPROVED** [review]

**Pull request merged**